creator.cpp

```diff
@@ -99,55 +99,55 @@ Creator::Creator(Privileges &privilegesArg, QWidget *parent) :
     // add minimize button on Windows
     this->setWindowFlags(windowFlags() | Qt::WindowMinimizeButtonHint);
 
-    connect(diskWriterThread, SIGNAL(finished()),
-            diskWriter, SLOT(deleteLater()));
-    connect(this, SIGNAL(proceedToWriteImageToDevice(QString,QString)),
-            diskWriter, SLOT(writeImageToRemovableDevice(QString,QString)));
-
-    connect(diskWriter, SIGNAL(bytesWritten(int)),this, SLOT(handleWriteProgress(int)));
-    connect(diskWriter, SIGNAL(syncing()), this, SLOT(writingSyncing()));
-    connect(diskWriter, SIGNAL(finished()), this, SLOT(writingFinished()));
-    connect(diskWriter, SIGNAL(error(QString)), this, SLOT(writingError(QString)));
+    connect(diskWriterThread, &QThread::finished,
+            diskWriter, &Creator::deleteLater);
```

Comment:
I guess the class should be `DiskWriter`

Reply:
Are you looking at the correct signal? We use `finished` twice, on line 102 and 109. In master branch, at line 102 we use `diskWriterThread`, and that's a `QThread`.
https://github.com/LibreELEC/usb-sd-creator/blob/10ef4dc831a58394875c518ddd2444c8f81b6159/creator.cpp#L102
When I made this commit, I didn't intend to introduce any functional change, or do you think it was wrong to begin with?

Reply:
I'm talking about slot :) sorry for not being clear